xapian-core/tests/testdata/apitest_cluster.txt

```diff
@@ -3,3 +3,13 @@ A computer cluster consists of a set of loosely or tightly connected computers t
 The components of a cluster are usually connected to each other through fast local area networks ("LAN"), with each node (computer used as a server) running its own instance of an operating cluster system. In most circumstances, all of the nodes use the same hardware and the same operating system, cluster although in some setups (i.e. using Open Source Cluster Application Resources (OSCAR)), different operating systems can be used on each computer, clusterand/or different hardware.
 
 They are usually deployed to improve performance and availability over that of a single computer, while typically being much more cost-effective than single computers of comparable speed or availability.
+
```

Comment:
This is taken from wikipedia, right?  Something like the first paragraph of each entry here it looks like:

https://en.wikipedia.org/wiki/Cluster

That's not an inherent problem, but the Wikipedia licence requires both attribution and making it clear what the licence is - see "Attribution" and "Notice" here:

https://en.wikipedia.org/wiki/Wikipedia:Text_of_Creative_Commons_Attribution-ShareAlike_3.0_Unported_License

If you only excerpted a small amount of text, one could argue that's not required, but this seems like a sizeable amount of text to me.